src/polymer/elements/rules.ts

```diff
@@ -14,3 +14,4 @@
 
 import './iron-flex-layout-classes';
 import './iron-flex-layout-import';
+import './paper-toolbar-v1-to-v2';
```

Comment:
Recommend removing the paper-toolbar special case code in content-to-slot-usages so that users don't see duplicated warnings/fixes

Reply:
Removed.